### PR TITLE
Ana/move dependabot config to root folder and add global lint command

### PR DESCRIPTION
**Pull request by Bitcoinera**
Move dependabot config to root folder, since it won't have any effect in the children directories

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Why is it triggering the checkChangelogs in API and extension? I think I did something wrong

**Comment by Bitcoinera** [comment]
Why it says that there are no changes pending? I also do it locally and the same. Strange

**Comment by Bitcoinera** [comment]
Extension testing /test is failing because node-modules are being installed in the root directory 👀

File: app/changes/ana_move-dependabot-config-to-root-folder [hunk 1]
```diff
@@ -0,0 +1 @@
+[Code Improvements] [#4216](https://github.com/cosmos/lunie/pull/4216) Move dependabot config to root folder of monorepo @Bitcoinera
```
Comment (faboweb) [inline app/changes/ana_move-dependabot-config-to-root-folder:1]: why did you copy this in app as well?

File: app/src/components/notifications/PageNotifications.vue [hunk 2]
```diff
@@ -121,7 +121,7 @@ export default {
       update(result) {
         this.dataLoaded = true
         // assume that when the full page got loaded, that there is more
-        this.moreAvailable = (result.notifications.length % 20 === 0)
+        this.moreAvailable = result.notifications.length % 20 === 0
```
Comment (faboweb) [inline app/src/components/notifications/PageNotifications.vue:124]: @michielmulders linting seems to kill the syntax

File: package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "changelog:api": "cd api && yarn changelog",
     "changelog:app": "cd app && yarn changelog",
-    "changelog:extension": "cd extension && yarn changelog"
+    "changelog:extension": "cd extension && yarn changelog",
+    "lint:global": "cd api && yarn lint --fix && cd .. && cd app && yarn lint --fix && cd .. && cd extension && yarn lint:fix && cd .."
```
Comment (faboweb) [inline package.json:19]: do we really need this? will you want to lint all repos at once?

File: app/changes/ana_move-dependabot-config-to-root-folder (hunk 1 above)
Comment (faboweb) [reply app/changes/ana_move-dependabot-config-to-root-folder:1]: I see.

> We should have some kind of global changelog 🤔 ? Idk, adding changelogs for all subrepos for now

yep, will think about an update to simsala

**Comment by faboweb** [comment]
This feels wrong:
image

**Comment by faboweb** [comment]
> Extension testing /test is failing because node-modules are being installed in the root directory 👀

extension CI is not finished yet

File: app/changes/ana_move-dependabot-config-to-root-folder (hunk 1 above)
Comment (Bitcoinera) [reply app/changes/ana_move-dependabot-config-to-root-folder:1]: Great stuff! 🤩 Simsala will now be monorepo-compatible

File: package.json (hunk 3 above)
Comment (Bitcoinera) [reply package.json:19]: yeah, why not. In global changes

File: app/changes/ana_move-dependabot-config-to-root-folder (hunk 1 above)
Comment (Bitcoinera) [reply app/changes/ana_move-dependabot-config-to-root-folder:1]: egleniyormuyuzcom-cilBKvPHOrWHO7YCAs

File: package.json (hunk 3 above)
Comment (Bitcoinera) [reply package.json:19]: How come that this is merged and I don't have it when I do `git pull`?

Comment (Bitcoinera) [reply package.json:19]: ah you revert it at some point right?

**Pull request merged**